docs/user-guide/backend/docker.md

```
@@ -1,56 +1,258 @@
+
+首次启动时，Docker Desktop 会要求您登录 Docker Hub 账号。如果您没有账号，需要先注册一个。
+
+### 获取 Docker 镜像
+
+点击顶部菜单的 **Search** 图标（或使用快捷键 `Ctrl + K`(Windows)或者`Cmd + K`(MacOS)）。
+
+![alt text](/docker_img/click_search.png)
+
+在搜索框中输入以下内容，搜索到以后点击右侧的**Pull**按钮。
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)